### PR TITLE
SASS-2001 - Delete Employment audit event.

### PR DESCRIPTION
**Pull request by georgievh**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-2001](https://jira.tools.tax.service.gov.uk/browse/SASS-2001)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

**Comment by matt712** [comment]
uk.gov.hmrc: bootstrap-frontend-play-28 and uk.gov.hmrc: bootstrap-test-play-28 could be bumped to 5.19.0

File: app/audit/DeleteEmploymentAudit.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                                 expenses: Option[Expenses]) {
+
+  private def name = "DeleteEmployment"
+
+  def toAuditModel: AuditModel[DeleteEmploymentAudit] = {
+    val maybeExpenses = expenses.map(_.copy(
+      businessTravelCosts = None,
+      hotelAndMealExpenses = None,
+      vehicleExpenses = None,
+      mileageAllowanceRelief = None
```
Comment (matt712) [inline app/audit/DeleteEmploymentAudit.scala:39]: Why are the expenses set to None here

**Review by matt712 — COMMENTED** [review]
Just one question about the DeleteEmploymentAudit file

File: app/audit/DeleteEmploymentAudit.scala (hunk 1 above)
Comment (georgievh) [reply app/audit/DeleteEmploymentAudit.scala:39]: That is the same as in ViewEmploymentExpensesAudit. I spoke to Ben at that point and I think it was due to fields being ignored.

**Review by matt712 — APPROVED** [review]

File: app/audit/DeleteEmploymentAudit.scala (hunk 1 above)
Comment (matt712) [reply app/audit/DeleteEmploymentAudit.scala:39]: I just confirmed this with Ben. 👍

**Review by matt712 — APPROVED** [review]

**Pull request merged**